documentation/markup.rst

```diff
@@ -1035,8 +1042,12 @@ units as well as normal text:
 
    Example::
 
-      .. versionchanged:: 3.1
-         The *spam* parameter was added.
+      .. function:: func(spam)
+
+         Return foo and bar with *spam* applied.
```

Comment:
```suggestion
      .. function:: func(spam=False)

         Return foo and bar, optionally with *spam* applied.
```

Be a little more realistic and consistent with the previous (avoiding a compat break, which in this case would usually mean a new function).